templates/ssm-parameters.j2

```diff
@@ -0,0 +1,46 @@
+# This is a cloudformation jinjaized template to install parameters into the
+# SSM parameter store.  This is only for non-securestring parameters.
+# Cloudformation does not support adding securestring parameters into the SSM.
+#
+# Usage:
+#  Create a sceptre template `my-ssm-params.yaml`
+#
+#  template_path: remote/ssm-parameters.j2
+#  hooks:
+#    before_launch:
+#      - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/master/ssm-parameters.j2 --create-dirs -o templates/remote/ssm-parameters.j2"
+#  stack_name: my-ssm-vars
+#  sceptre_user_data:
+#    Prefix: /my-ssm-vars/
+#    Parameters:
+#      - Name: param1
+#        Value: foo
+#      - Name: param2
+#        Value: foo
+
+Description: "Setup parameters in the SSM parameter store"
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  {% for parameter in sceptre_user_data.Parameters %}
+  {%- set name = parameter.Name -%}
+  {%- set value = parameter.Value -%}
+    {{ name }}:
+      Type: AWS::SSM::Parameter
+      Properties:
+        Name: {{ sceptre_user_data.Prefix }}{{ name }}
```

Comment:
would it be more consistent stylistically to add a line `{%- set prefix = sceptre_user_data.Prefix -%}` (outside the for-loop) and then change this to `Name: {{ prefix }}{{ name }}`?

Reply:
Or even better: `Name: "{{ prefix }}{{ name }}"` ?

Reply:
no that won't work, it needs to be `sceptre_user_data.Prefix`.  again, template is only being moved, it's already working as is.

Reply:
Not sure I understand why that wouldn't work, but my comment was just stylistic anyway.

Reply:
it wouldn't work because `Prefix` is not in the `sceptre_user_data.Parameters` dict.  jinja requires a full reference to `Prefix` so it needs to be `sceptre_user_data.Prefix`.